addons/markers/functions/fnc_isMarkerOwner.sqf

```diff
@@ -0,0 +1,30 @@
+#include "script_component.hpp"
+/*
+ * Author: McDiod
+ * Checks whether player created the passed marker.
+ *
+ * Arguments:
+ * 0: Marker <String>
+ *
+ * Return Value:
+ * Whether the player created the marker. <BOOL>
+ *
+ * Example:
+ * [_marker] call ace_markers_fnc_isMarkerOwner
+ *
+ * Public: No
+ */
+
+params [["_marker",""]];
+
+// user has never created a marker
+if (isNil QGVAR(directPlayID)) exitWith {false};
+
+if !((_marker find "_USER_DEFINED") isEqualTo 0) exitWith {false};
```

Comment:
This is equivalent to:
`_marker find "_USER_DEFINED" != 0`
I think. Isn't that easier to read as well as faster?

Why 0 and not -1?

Reply:
Some joker might call his editor marker "myMarker_USER_DEFINED", unlikely but possible. I don't mind if you prefer -1 though.

Reply:
I don't get how 0 vs -1 helps in that case.

Reply:
0 vs -1:
```sqf
// not a user defined marker
_marker = createMarker ["myMarker_USER_DEFINED",[0,0,0]];

// true
!((_marker find "_USER_DEFINED") isEqualTo 0)

// false
((_marker find "_USER_DEFINED") isEqualTo -1)
```

As for the `!=` vs. `isEqualTo`: Can be changed. I just used isEqualTo to be in line with
https://github.com/acemod/ACE3/blob/master/addons/markers/functions/fnc_initInsertMarker.sqf#L58

Reply:
How are these different from a mission created marker? I.e. `57298579257_USER_DEFINED`?

Reply:
Mission created markers are `_USER_DEFINED #712781623786/2/5` or something.

Reply:
This does not make any sense to me.
>Why 0 and not -1?
>Some joker might call his editor marker "myMarker_USER_DEFINED", unlikely but possible.

They could just name it `_USER_DEFINED 123`. My question is still not answered.